core/scale/compact.cpp

```diff
@@ -14,7 +14,7 @@
 #include "scale/scale_error.hpp"
 #include "scale/util.hpp"
 
```

Comment:
so many places where common::Buffer makes formatting worse. Maybe add using common::Buffer; here?

Reply:
I don't see a problem to write common::Buffer

Reply:
if it's fine for you, then I don't mind.